src/Tizen.Wearable.CircularUI.Forms/GoogleMapOption.cs

```diff
@@ -0,0 +1,148 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Xamarin.Forms.Maps;
+
+namespace Tizen.Wearable.CircularUI.Forms
+{
+    /// <summary>
+    /// The GoogleMapOption struct used to define the properties that can be set on a MapView.
+    /// </summary>
+    public struct GoogleMapOption
+    {
+        private double _zoom;
+
+        public GoogleMapOption(Position center, GoogleMapType type = GoogleMapType.Roadmap, int zoomLevel = 10, 
+                bool visibleZoomControl = false, bool pinsPopupOpened = false, bool enableGesture = true, 
+                ZoomControlPosition controlPosition = ZoomControlPosition.RightBottom )
+        {
+            Center = center;
+            MapType = type;
+            _zoom = Math.Min(Math.Max(zoomLevel, 1.0), 20.0); ;
+            IsVisibleZoomControl = visibleZoomControl;
+            IsPinsPopupOpened = pinsPopupOpened;
+            IsEnableGestureHandle = enableGesture;
+            ZoomControlPosition = controlPosition;
+        }
+
+        /// <summary>
+        /// Gets or sets a Google Map type value of GoogleMapOption.
+        /// </summary>
+        public GoogleMapType MapType { get; set; }
+
+        /// <summary>
+        /// Gets or sets a center postion of GoogleMapOption.
+        /// This value set center of MapView. But getting value not mean current center of MapView because value can't reflect user interaction.
+        /// </summary>
+        public Position Center { get; set; }
+
+        /// <summary>
+        /// Gets or sets a zoom level of GoogleMapOption.
+        /// This value set zoom level of MapView. But getting value not mean current zoom level of MapView because value can't reflect user interaction.
+        /// </summary>
+        public double Zoom 
+        {
+            get { return _zoom; }
+            set {
+                if (value > 20.0)
+                    _zoom = 20;
+                else if(value < 1.0)
+                    _zoom = 1;
+                else
+                    _zoom = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets a boolean value that indicates whether zoom control is visible.
+        /// </summary>
+        public bool IsVisibleZoomControl { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean value that indicates whether zoom and pan interaction is enable.
+        /// </summary>
+        public bool IsEnableGestureHandle { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean value that indicates whether Pins popup is opened.
+        /// </summary>
+        public bool IsPinsPopupOpened { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean value that indicates whether zoom and pan interaction is enable.
+        /// </summary>
+        public ZoomControlPosition ZoomControlPosition { get; set; }
```

Comment:
What about `MapControlPositon`?

Reply:
ZoomControlPosition  is more clear than MapControlPositon. each Map control can place separately.